common/src/main/java/org/apache/uniffle/common/util/NettyUtils.java

```diff
@@ -43,6 +43,13 @@ public class NettyUtils {
 
   private static final long MAX_DIRECT_MEMORY_IN_BYTES = PlatformDependent.maxDirectMemory();
 
+  /** Specifies an upper bound on the number of Netty threads that Uniffle requires by default. */
+  private static int MAX_DEFAULT_NETTY_THREADS = 8;
```

Comment:
Why 8 as the upper bound?

Reply:
The codes are copied from Spark, useless comments are removed. Although it says upper bound, but actually it is a way to ensure **_minimum_** concurrency. 
Refer to https://github.com/apache/spark/blob/master/common/network-common/src/main/java/org/apache/spark/network/util/NettyUtils.java#L50.

Reply:
Got it.